src/main/java/org/apache/cassandra/sidecar/utils/FileStreamer.java

```diff
@@ -123,6 +128,8 @@ private void acquireAndSend(HttpResponse response,
                                {
                                    LOGGER.debug("Streamed file {} successfully to client {}. Instance: {}", filename,
                                                 response.remoteAddress(), response.host());
+                                   metricRegistry.register(MetricRegistry.name(FileStreamer.class, "bytes_streamed"),
+                                                     (Gauge<Long>) () -> fileLength);
```

Comment:
It does not work as expected. 
The code here tries to register the Gauge metric whenever a file streams successfully and only the first attempt succeeds. It then, only publishes the `fileLength` of that file and once. 

It should not use `Gauge` type. Instead, it should be `Histograms` to not loose any update.